pants.ini

```diff
@@ -180,8 +180,6 @@ exclude_patterns: [
 [compile.zinc]
 jvm_options: [
     '-Xmx4g', '-XX:+UseConcMarkSweepGC', '-XX:ParallelGCThreads=4',
-    # bigger cache size for our big projects (default is just 5)
-    '-Dzinc.analysis.cache.limit=1000',
```

Comment:
How come we don't need this any more?

Reply:
I changed the default in the JVM code: because it's a weak-reference cache, making it huge is safe.